python/grass/jupyter/interactivemap.py

```diff
@@ -394,9 +397,12 @@ def handle_interaction(**kwargs):
                     (reprojected_coordinates[0], reprojected_coordinates[1]),
                     self.raster_name,
                 )
+                self.region = get_region(env=os.environ.copy())
+
                 vector_output = query_vector(
                     (reprojected_coordinates[0], reprojected_coordinates[1]),
                     self.vector_name,
+                    10.0 * ((self.region["east"] - self.region["west"]) / self.width),
```

Comment:
This should not be the computational region, but rather the visible extent which would change based on how you zoom.

Reply:
I couldn't understand what computational region means here; we are just setting the distance. Could you please brief me about this?

Reply:
The distance needs to be based on the extent of the map, so for example when you zoom out a lot, when you click at the vector point, the distance between the place you clicked and the actual coordinates of the vector point may be several kilometers. When you zoom in, the distance can be for example, several meters.

Reply:
I tried using `distance = 10.0 * (self.region["east"] - self.region["west"])/self.map.zoom`, but it gives me the same vector value at all points in the map. Could you please help me with this?

Reply:
You can use m.bounds to get the extent of the leaflet map and then the width in pixels you know, because it's an input to InteractiveMap. You would need to reproject the bounds.
If this is too difficult, we can also use the computational region as you have it now. Not the best, but perhaps sufficient for now. But it needs to be noted there, that the approach can be improved in the future.

Reply:
Hi Anna, I was trying to reproject coordinates, but I've been stuck on it for three days. I am taking the bounds using `m.bounds` and then calculating the latitude and longitude as:
                `central_latitude = (north + south) / 2`
                `central_longitude = (east + west) / 2`
I pass this latitude and longitude to reproject_latlon, which gives me the east and north. I was looking for some way to get the east and west, but I need something to work out.

Reply:
Why central point? Just use lower left and top right corner and reproject those. No need to get stuck on this so long, as I suggested, it's OK to move on.